assets/app/index.html

```diff
@@ -124,14 +127,21 @@
         <script src="scripts/controllers/images.js"></script>        
         <script src="scripts/controllers/deployments.js"></script>
         <script src="scripts/controllers/services.js"></script>
+        <script src="scripts/controllers/create/createFromImage.js"></script>
         <script src="scripts/controllers/newfromtemplate.js"></script>
         <script src="scripts/controllers/labels.js"></script>
         <script src="scripts/controllers/tasks.js"></script>
         <script src="scripts/controllers/util/oauth.js"></script>
         <script src="scripts/controllers/util/error.js"></script>
         <script src="scripts/controllers/util/logout.js"></script>
-        <script src="scripts/controllers/catalog.js"></script>
+        <script src="scripts/controllers/catalog/templates.js"></script>
+        <script src="scripts/controllers/catalog/images.js"></script>
+        <script src="scripts/controllers/create.js"></script>
         <script src="scripts/directives/date.js"></script>
+        <script src="scripts/directives/oscFormSection.js"></script>
```

Comment:
If we do decide to prefix the files, it'd make sense to eventually prefix all of them